docs/browser/events.md

```diff
@@ -0,0 +1,158 @@
+# Semantic conventions for browser events
+
+**Status**: [Experimental](../../../document-status.md)
+
+<details>
+<summary>Table of Contents</summary>
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [PageView](#pageview)
+- [PageNavigationTiming](#pagenavigationtiming)
+- [ResourceTiming](#resourcetiming)
+- [Exception](#exception)
+- [UserAction](#useraction)
+- [WebVital](#webvital)
+
+<!-- tocstop -->
+
+</details>
+
+This document describes the semantic conventions for events on browser platform. All browser events MUST use a namespace of `browser` in the `event.name` property.
+
+## PageView
+
+The event name MUST be `browser.page_view`.
+
+This event describes the details of the web page visited.
+
+The following table describes the payload fields that MUST be used to describe the details of event.
+
+| Body Field  | Type | Description  | Examples  | Requirement Level |
```

Comment:
not blocking: do we envision reusing attribute requirement levels? Would we need all levels?

Reply:
@MSNev can you please help respond to this comment? thanks.

Reply:
Yes